compiler/x/codegen/OMRTreeEvaluator.hpp

```diff
@@ -300,7 +300,9 @@ class OMR_EXTENSIBLE TreeEvaluator: public OMR::TreeEvaluator
    static TR::Register *mLongBitsToMaskEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *mRegLoadEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *mRegStoreEvaluator(TR::Node *node, TR::CodeGenerator *cg);
-
+   static TR::Register *mandEvaluator(TR::Node *node, TR::CodeGenerator *cg);
+   static TR::Register *morEvaluator(TR::Node *node, TR::CodeGenerator *cg);
+   static TR::Register *mxorEvaluator(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
A nitpick: There is an empty line between `mxorEvaluator()` and `b2mEvaluator()` on other platforms.